docs/content/docs/test/FlakyTests.md

```diff
@@ -1,53 +1,58 @@
+- Игнорировать тесты в TeamCity. Вы просто "подложите свинью" своим коллегам в будущем.
+- Помечать тест как `@Ignore`. Есть специальная аннотация [`@Flaky`]({{< ref "/docs/test/FlakyAnnotation.md" >}}).
 
-That's why we introduce annotion to mark your test flaky. 
+## Убедись что тест действительно флакует
 
-We want:
-- Minimize flakiness
-- Launch as many tests as we can on pull request checks
-- Help to fix common flaky problems
+1. Посмотри в чаты [#android-dev](http://links.k.avito.ru/slackandroiddev) или 
+   [#android-autotesting](http://links.k.avito.ru/slackandroidautotesting). Вероятно, проблема уже известна.
+1. Посмотри [статистику стабильности теста]({{< ref "#где-посмотреть-статистику-по-стабильности-тестов" >}})
+1. Запусти тест для проверки в несколько прогонов в [динамической конфигурации]({{< ref "/docs/test/Run.md#custom-run" >}}).
 
-## `@Flaky` annotation
+## Отлаживай в IDE
 
-Such tests are marked with special annotation - `@Flaky`. You also able to provide an optional reason of flakiness for this test.
+В Android Studio должен из коробки работать debug на конкретном тесте.
 
-```kotlin
-@Flaky(reason = "Relies on real back-end")
-class MyAwesomeTests {
-    // ...
-}
-```
+https://developer.android.com/studio/debug
 
-This annotation can be added both to the whole class and to a separate test:
+### Layout Inspector
 
-```kotlin
-class MyAwesomeTests {
-    @Flaky
-    @Test
-    fun foobar() {
-        // ...
-    }
-}
-```
+<br> *Layout inspector использует adb поэтому мы не можем получить состояние экрана в дебажном запуске.* \
+<br> __Как получить экран во время прогона теста?__
+1. Добавить в нужное место `Thread.sleep()`.
+2. Запустить тест без дебага и дождаться пока исполнение попадет в `Thread.sleep()`
+3. Задампить состояние экрана через Layout Inspector
 
-## Suppress `@Flaky` test failures
+## Если не удалось найти причину
 
-To suppress failures of @Flaky tests just add `suppressFlaky = true` to your `uiTests` step.
-For more information on configuring build steps for the CI Steps plugin, please refer to [this document]({{< ref "/docs/projects/CISteps.md" >}}).
+1. Обратись к автору тесту или юниту, который за него отвечает.
+1. Если тест уже чинят, то можно дождаться фикса.
+1. Если проблема новая, то договоритесь о том, кто поставить аннотацию `@Flaky` на тест.
+1. В аннотации обязательно укажите причину, а лучше задачу, в которой тест будет исправлен.
+1. Запушьте изменения
+
+## Если проблема никак не решается
+
+1. [Обратись за помощью]({{<ref "/docs/Contacts.md" >}})
+1. Приложи сылку на репорт.
+1. Помогут также логи с ошибками.
+
+## Где посмотреть статистику по стабильности тестов?
+
+- [Общая статистика по тестам (internal)](http://links.k.avito.ru/FR)
+- [История нестабильности теста (internal)](http://links.k.avito.ru/5W)
+
+Обратите внимание на параметры фильтрации.
+
+## Когда убирать аннотацию `@Flaky`
+
+Если вы внесли изменения, которые чинят исходную причину флакования теста, то в том же PR 
```

Comment:
Also, we could provide an option to look at BI statistic board

Reply:
Added reference to previous section

Reply:
I only want that to become a more direct recommendation. Use BI when you want to remove `Flaky`